rector.yml

```diff
@@ -2,7 +2,6 @@ parameters:
   sets:
     - 'code-quality'
     - 'dead-code'
-    - 'php72'
```

Comment:
can you also reactivate these? I wasnt able to make a suggestion for them

Reply:
Fixed!